completions/chronyc

```diff
@@ -4,7 +4,7 @@ _comp_cmd_chronyc__command_args()
 {
     local -a args
     _comp_split args "$("$1" help 2>/dev/null |
-        awk '/^'"$prev"'\s[^ []/ { gsub("\\|", " ", $2); print $2 }')"
+        command awk '/^'"$prev"'[ 	][^ []/ { gsub("\\|", " ", $2); print $2 }')"
```

Comment:
I wonder if there's a better way to express this, the literal tab isn't quite obvious just by reading.

Reply:
I'll replace it with `\t`. Although POSIX ERE does not define `\t`, POSIX awk specifies the extension to the ERE for escape sequences of control characters including `\t`.

Reply:
Fixed it in https://github.com/scop/bash-completion/pull/1072/commits/1989ba9ad3de5cf33da3c9623fa87be5194fac08#diff-24150d0a0f171f44ed1383b6c2dff5a662fdee266f192390f67c3e26c22a4c42